### ISSUE TITLE
hsla editor lockup on quick update

### ISSUE THREAD

**Issue by atanasster** [omgovich/react-colorful#76]
Hi, Thanks very much for the component, react-color was indeed very heavy.

I have integrated it already and would be great if you can list it as one of the projects using react-colorful: https://component-controls.com/api/editors-coloreditor--overview

I seem to have an issue with fast updating of the control values with the hsla (and rgba) color picker (the hex color picker seems fine), any ideas what could be going on?

Here is a video, from the page linked above and the control locking up on fast moving around:

![react-colorful](https://user-images.githubusercontent.com/6075606/96615643-8db65880-12cf-11eb-81fe-799c4e3110ce.gif)

Also I resorted to converting rgba/hsla values since the HSLAStringColorPicker and rgba equivalent are not rounding the values, so I could end up with lets say 39.999999999 a one of the values. Is there an option for rounding the values (in my case I need to round to 0 decimals for the colr values and 2 decimals for the alpha value)

**Comment by omgovich** [comment]
Hi! What a nice project you have! Will do my best to help you.

Rounding values is one of the thing I'm thinking of. Agree with you. We definitely should do that.
Will try to implement it this week.

**Comment by omgovich** [comment]
Give me some time to fix the rounding issue. I assume it fixes the component locking issue as well.

**Comment by atanasster** [comment]
Thanks a lot, yes i am sure the rounding option will fix the other issue as well.

Thanks for appreciating my project - can I contribute a documentation/testing site for react-colorful?

**Comment by omgovich** [comment]
Hey @atanasster.

We just released `v4.2.0` that rounds all output values.

Try to upgrade `react-colorful` in your project and use string color pickers:
```diff
import {
  HexColorPicker,
-  HslaColorPicker,
-  HslColorPicker,
-  RgbColorPicker,
-  RgbaColorPicker,
+  HslaStringColorPicker,
+  HslStringColorPicker,
+  RgbStringColorPicker,
+  RgbaStringColorPicker,
} from 'react-colorful';
```

**Comment by atanasster** [comment]
@omgovich fantastic, works great
